ts/src/bingx.ts

```diff
@@ -389,6 +388,7 @@ export default class bingx extends Exchange {
                                 'uid': 1,
                                 'apiKey/query': 2,
                                 'account/apiPermissions': 5,
```

Comment:
@ttodua hum I had the idea I fixed this the other day

Reply:
I was also mistaken. commented here too, and now this implicit api works (tested too): https://t.me/ccxt_chat/16693

Reply:
@ttodua I was referring to this PR: https://github.com/ccxt/ccxt/pull/25575/files

Reply:
![image](https://github.com/user-attachments/assets/5d8ddf33-8e3b-415a-8fd1-1e30a4a67bd1)

Reply:
ouch man, whoaat

does github still do the glitch it had an year ago or so?

Reply:
@ttodua yeah what the heck 😅  my changed is gone and the file history does not explain how

Reply:
Don't merge this PR for a while.
we should investigate this, maybe contact github to know insights.
did you or Igor contact github in the past, on this same issue? maybe we can update that ticket too

Reply:
i've gone through the commits history since your PR merger, and there was no further commit/push for bingx, and how it become reverted? something bug unless we miss something